backend/db/utils.py

```diff
@@ -156,14 +156,24 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
 
-def update_db_status_var(key: str, val: str, local=False):
+def delete_db_status_var(key: str, local=False):
```

Comment:
Thanks for adding this. Just a couple cleanup steps:
- Move this todo `todo: change to a 1-liner UPDATE statement` from `delete_db_status_var()` into the `update_db_status_var()` docstring
- Update the `delete_db_status_var()` docstring to something like "Delete a variable from the `manage` table.

Reply:
Confirmed these changes are made, thanks Hope!